frontend/src/components/common/SimpleTable.tsx

```diff
@@ -104,6 +112,34 @@ export default function SimpleTable(props: SimpleTableProps) {
   // eslint-disable-next-line
   [data, currentData]);
 
+  React.useEffect(() => {
+    if (data) {
+      let applySort = undefined;
+      const columnAskingForSort = columns.find((col) => col.label === sortObj.label);
+      const sortFunction = columnAskingForSort?.sort;
+
+      if (typeof sortFunction === 'boolean' && sortFunction){
+        if (sortObj.isIncreasingOrder){
+          setCurrentData(data.sort());
+        } else {
+          setCurrentData(data.sort().reverse());
```

Comment:
This function is very likely to be inneficient because at least in Headlamp will mostly be comparing classes.
So we should provide the default sorting function that will use the column's `getter`/`datum` and compare the results, i.e. if the getter is `getter: (item: Event) => item.name`, then we need to do something like (but with types :smile_cat: ):
```
function defaultSortingFunction(column) {
  function defaultSortingReal(item1, item2) {
    let value1 = column.getter(item1);
    let value2 = column.getter(item2);

    let compareValue = 0;
    if (value1 < value2) {
      compareValue = -1;
    } else if (value1 > value2) {
      compareValue = 1;
    }

    return compareValue * (isAscendingOrder ? 1 : -1);
  }

  return defaultSortingReal;
}

...

data.sort(defaultSortingFunction(col[colIndex]));